src/Pusher.php

```diff
@@ -117,7 +63,7 @@ public function __construct($auth_key, $secret, $app_id, $options = array(), $ho
             $this->settings['host'] = $host;
 
             $this->log('Legacy $host parameter provided: '.
-                                    $this->settings['scheme'].' host: '.$this->settings['host']);
+            $this->settings['scheme'].' host: '.$this->settings['host']);
```

Comment:
I'm also a bit unsure about the style here, as it reads as a new statement and not a continuation of the previous one.